augur/ancestral.py

```diff
@@ -1,5 +1,16 @@
 """
 Infer ancestral sequences based on a tree.
+
+The ancestral sequences are inferred using `TreeTime <https://academic.oup.com/ve/article/4/1/vex042/4794731>`_.
+Each internal node gets assigned a nucleotide sequence that maximizes a
+likelihood on the tree given its descendants and its parent node.
+Each node then gets assigned a list of nucleotide mutations for any position
+that has a mismatch between its own sequence and its parent's sequence.
+The node sequences and mutations are output to a node-data JSON file.
+
+.. note::
+
+    The mutation positions in the node-data JSON are one-based.
```

Comment:
These `.. note::` directives don't render well in `--help` output

```
Infer ancestral sequences based on a tree. The ancestral sequences are inferred
using `TreeTime <https://academic.oup.com/ve/article/4/1/vex042/4794731>`_. Each
internal node gets assigned a nucleotide sequence that maximizes a likelihood on
the tree given its descendants and its parent node. Each node then gets assigned a
list of nucleotide mutations for any position that has a mismatch between its own
sequence and its parent's sequence. The node sequences and mutations are output to
a node-data JSON file. .. note:: The mutation positions in the node-data JSON are
one-based.
```

but I think it's readable enough and fine for now. We can improve Augur's ability to have richer `--help` output in the future.

Alternatively, I think we could drop the directive and just put the sentence in the text without a call out.

Reply:
> We can improve Augur's ability to have richer --help output in the future.

Yeah, I was just looking at how you're handling the docs and --help integration in the Nextstrain CLI. I'll add to my to-do list to port it over to Augur when I want to work on docs again 😄 

> Alternatively, I think we could drop the directive and just put the sentence in the text without a call out.

I'm happy with leaving it as-is for now. I think it's nice to have the obvious call out in the docs since the one-based counting can trip people up.

Reply:
> I'll add to my to-do list to port it over to Augur when I want to work on docs again :smile:

Sounds good! It's a decently dense chunk of code, so if we're going to use it elsewhere it might be best as a separate library shared by both Nextstrain CLI and Augur.

> I'm happy with leaving it as-is for now. I think it's nice to have the obvious call out in the docs since the one-based counting can trip people up.

:+1: